build/tsconfig.docs.json

```diff
@@ -3,10 +3,7 @@
   "compilerOptions": {
     "module": "esnext",
     "paths": {
```

Comment:
`paths` can be removed there

Reply:
Can you please also check `include`. I don't think that we should include `react` anymore